internal/log/log.go

```diff
@@ -3,20 +3,56 @@
 package log
 
 import (
-	"github.com/sirupsen/logrus"
+	"bytes"
+	"fmt"
+
+	"github.com/go-logr/logr"
 )
 
-var l *logrus.Logger
+const (
+	DBG int = 1
+	TRC int = 2
+)
 
-// Logger returns the configured logger, or a new unconfigured logger
-// if one has not already been configured.
-func Logger() *logrus.Logger {
-	if l == nil {
-		l = logrus.New()
+func NewBufferSink(buffer *bytes.Buffer) logr.LogSink {
```

Comment:
Wow, we had to generate a net-new buffer sink? There was not an existing one we could have leveraged from the logr package?

Reply:
I didn't see one, but I also didn't look that hard. It took all of about 3 minutes to create.